smelli/classes.py

```diff
@@ -199,6 +217,15 @@ def _load_likelihoods(self,
                 yaml_dict = flavio.io.yaml.load_include(f)
                 if not self.fix_ckm:
                     yaml_dict['par_obj'] = par_ckm_dict
+                if fn in add_measurements.keys():
+                    yaml_dict['include_measurements'] += add_measurements[fn]
```

Comment:
I think it would be good to test for the presence of the added measurements at this point. Otherwise, measurements that are not present in `flavio` would lead to an error later in the code, telling the user to update `smelli`. However, if the user wants to add a measurement to the likelihood that is not in `flavio` yet, the error message should tell the user to add this measurement to `flavio`, not to update `smelli`.

Reply:
I defined the new exception as a `ValueError`. Is that OK, or should I change it to an `AssertionError`?

Reply:
I think `ValueError` is the right one here since it should be raise when the user input has an inappropriate value.

Reply:
From the Python docs: "`AssertionError`: Raised when an assert statement fails." So you should actually never raise this yourself.

Reply:
Well, at the moment, we actually have a place where we raise an `AssertionError`:
https://github.com/smelli/smelli/blob/550e2016f2f6042757bac4262b3c441705911cf7/smelli/classes.py#L207-L212
But in lines 207-209, an actual `AssertionError` is catched and then raised with an extended error message. So I think this should be fine. In lines 210-212 I agree that this is a bit of misuse of `AssertionError`. It is just used there because the error message is very similar to the other case in the lines above.

Reply:
Ha, interesting. Clearly, flavio should have raised a `ValueError` there and not an `AssertionError`. I guess the guy who wrote `FastLikelihood.load_dict` didn't know about [Python's `-O` option](https://docs.python.org/3/using/cmdline.html#cmdoption-O), which is a good reason *never* to use `assert` outside of unit tests.